[Tests]
  [./exception_test]
    type = 'RunException'
    input = 'exception.i'
    expect_err = "Caught exception"
    petsc_version = '>=3.1.0'
    should_crash = false
    max_threads = 1
    deleted = 'refs #2225'
  [../]

  [./parallel_exception_residual]
    type = 'RunException'
    input = 'parallel_exception_residual.i'
    expect_err = "Caught exception 2"
    petsc_version = '>=3.1.0'
    min_parallel = 2
    should_crash = false
    compiler = 'GCC INTEL'
    max_threads = 1
    deleted = 'refs #2269'
  [../]

  [./parallel_exception_initial_condition]
    type = 'RunException'
    input = 'parallel_exception_initial_condition.i'
    expect_err = "Caught exception 1"
    petsc_version = '>=3.1.0'
    min_parallel = 2
    should_crash = false
    compiler = 'GCC INTEL'
    max_threads = 1
    deleted = 'refs #2269'
  [../]

  [./parallel_exception_jacobian]
    type = 'RunException'
    input = 'parallel_exception_jacobian.i'
    expect_err = "Caught exception 3"
    petsc_version = '>=3.1.0'
    min_parallel = 2
    should_crash = false
    compiler = 'GCC INTEL'
    max_threads = 1
    deleted = 'refs #2269'
  [../]
[]